### DISCUSSION TITLE
Czy bilansowanie godzinowe działa? Jeżeli tak to w jakich okolicznościach?

### DISCUSSION THREAD

**Discussion by travnick** [PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus#151] [Q&A]
Chciałem prześledzić aktualną sytuację z energią na liczniku i wygląda na to, że dodatek nie bilansuje godzinowo energii pobranej i oddaną.

Mam zaznaczona opcję "Obliczanie bilansowania godzinowego", ale w zasadzie nie widzę różnicy w wartościach czy jest ona zaznaczona, czy nie.

Dla przykładu odczyt z bieżącego miesiąca:

portal tauron-elicznik (pełne dane z 1.06.2023 i częściowe - do godziny 15 - z 02.06.2023):
- pobrano: 10,2kWh
- oddano: 52,3kWh

wtyczka HA:
- Daily balance: -21,864 kWh (jak widzę to zwykła różnica energii pobranej i oddanej)
- Daily energy consumption: 7,599 kWh
- Daily energy generation: 29,463 kWh
- Monthly energy balance: -21,864 kWh (co to za wartość i czemu równa różnicy dziennej?)
- Monthly energy consumption: 10,225 kWh
- Monthly energy generation: 52,329 kWh

Czy to działa? Jak działa? kiedy działa? :D

**Comment by PiotrMachowski** [comment]
> jak widzę to zwykła różnica energii pobranej i oddanej

Stan efektywnie tak, ale tutaj wartości, które Cię interesują są w atrybutach.

> co to za wartość i czemu równa różnicy dziennej?

Jest to wartość dla wszystkich dni aktualnego miesiąca, dla których są pełne dane. Skoro w eLiczniku masz tylko jeden dzień, to to jest wartość dla właśnie niego.
